server/website/website/tasks/periodic_tasks.py

```diff
@@ -92,18 +90,11 @@ def run_background_tasks():
         LOG.debug("Aggregated knob data: rowlabels=%s, columnlabels=%s, data=%s.",
                   len(knob_data['rowlabels']), len(knob_data['columnlabels']),
                   knob_data['data'].shape)
+        LOG.debug("Aggregated metric data: rowlabels=%s, columnlabels=%s, data=%s.",
+                  len(metric_data['rowlabels']), len(metric_data['columnlabels']),
+                  metric_data['data'].shape)
         LOG.info("Done aggregating data for workload %s.", workload_name)
 
-        num_valid_results = knob_data['data'].shape[0]  # pylint: disable=unsubscriptable-object
-        if num_valid_results < MIN_WORKLOAD_RESULTS_COUNT:
-            # Check that there are enough valid results in the workload
-            LOG.info("Not enough valid results in workload %s (# valid results: "
-                     "%s, # required: %s).", workload_name, num_valid_results,
-                     MIN_WORKLOAD_RESULTS_COUNT)
-            workload.status = WorkloadStatusType.PROCESSED
```

Comment:
Same question here as above. Why do we no longer need this check?

Reply:
Sorry. Something is wrong during rebase. Those lines are now restored.